src/portfoliomgr/template_filters.py

```diff
@@ -1,6 +1,5 @@
 from django.template.defaulttags import register
-import json
-import os
+from common.views import current_app_version
```

Comment:
change the include once you do the change above

Reply:
Changes made